docs/ref-manual/cli/options.md

```diff
@@ -119,8 +119,10 @@ When you have a rule with multiple assertions:
 (--rule_sanity)=
 ### `--rule_sanity`
 
-**What does it do?**  
-This mode will check for each rule that even when ignoring all the user-provided assertions, the end of the rule is reachable. Namely, that the combination of requirements does not create an “empty” rule that is always true.
+**What does it do?**
+This mode will do some sanity checks for each rule, based on the attached value, which is allowed to be one of the following: `none`, `basic`, `advanced`.
+There are 3 kinds of sanity checks:
+1. Reachability- checks that even when ignoring all the user-provided assertions, the end of the rule is reachable. Namely, that the combination of requirements does not create an “empty” rule that is always true.
```

Comment:
A few formatting comments:
 - the subsequent paragraphs of a numbered list entry need to be indented 4 spaces.  Otherwise, they are not considered part of the list, so the next numbered section restarts at 1 (in the [generated documentation](https://certora-certora-prover-documentation--17.com.readthedocs.build/en/17/docs/ref-manual/cli/options.html#rule-sanity) there are two bullet 1s).  See https://www.markdownguide.org/basic-syntax/#adding-elements-in-lists
 - you should use triple backticks followed by the name of the language for multi-line code blocks.  This will give proper indentation and syntax highlighting.  In this case, you want
````
```cvl
rule empty_rule() {
  ...
}
```
````